portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase

```diff
@@ -65,4 +65,54 @@ definition {
 		}
 	}
 
+	@description = "StructuredContent can be filtered with a set priority"
+	@priority = "5"
+	test StructuredContentContainsUpdatedPriorityValueFilteredByEqualTo {
+		property portal.acceptance = "true";
+
+		task ("Given three web contents are created with default priority") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
+
+			var responseToParse1 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 1");
+			var responseToParse2 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 2");
+			var responseToParse3 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 3");
+		}
+
+		task ("When edit a web content with different priorities") {
+			HeadlessWebcontentAPI.editStructuredContentWithPriority(
```

Comment:
This step can be a Given too, as it is a precondition of what we want to check, that is: filter by priority after having edited it's value :)